scripts/build_package.sh

```diff
@@ -34,7 +34,7 @@ fi
 
 UNAME=$(uname)
 if [[ "${UNAME}" == *"MINGW"* ]]; then
-	GOPATH1=$HOME/go
+	export GOPATH=$HOME/go
```

Comment:
I think that there are three issues with this if statement:
1. we shouldn't compare here against MINGW; instead, we should use the `scripts/ostype.sh` to determine OS and use that.
2. the fact the we always update the GOPATH here is likely to be wrong. We definitely want to export it, but the caller should have set it's value already.
3. the fact that we have a difference between the windows and linux here suggest that something isn't aligned.. we should align the behavior and expectations.

Reply:
1. May be better although Windows is the only different than the rest. Wll have to patch several places.
2. Yes but the problem is `MSYS2` package only sets `GOROOT`. I think `GOPATH` is not added because Go is installed globally.
3. Windows is a totally different world, actually the usage of `MSYS2` is to avoid converting all the scripts to batch or powershell. But `MSYS2` also has its limitations, that's why these sort of patches.